example/api-executor/README.md

```
@@ -0,0 +1,171 @@
+});
+
+const source = streamResp?.$response?.data ?? streamResp;
+
+for await (const item of parseNDJSONStream(source)) {
+  console.log(item.result.object);
+}
+```
+
+## Expected Output
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)